experimental/framework/data-objects/src/signaler/signaler.ts

```diff
@@ -7,15 +7,14 @@ import { EventEmitter } from "events";
 import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
 import { TypedEventEmitter } from "@fluid-internal/client-utils";
 import { assert } from "@fluidframework/core-utils";
-import { Jsonable } from "@fluidframework/datastore-definitions";
 import { IInboundSignalMessage } from "@fluidframework/runtime-definitions";
 import { IErrorEvent } from "@fluidframework/core-interfaces";
 
 // TODO:
 // add way to mark with current sequence number for ordering signals relative to ops
 // throttling and batching
 
-export type SignalListener = (clientId: string, local: boolean, payload: Jsonable) => void;
+export type SignalListener = (clientId: string, local: boolean, payload: any) => void;
```

Comment:
This one I would convert.  It's experimental, so not critical...  but also a low-risk opportunity to exercise the new type.

Reply:
Lowish risk but plenty of overhead. The lack of typing in the signal space is how I started my journey. There isn't any association of specific signals and their payloads. If these were changed to `JsonableType` then all existing listeners will be dealing with that type and not `any`. For example, in example https://github.com/microsoft/FluidFramework/blob/main/examples/apps/presence-tracker/src/FocusTracker.ts:
```typescript
	private readonly onFocusSignalFn = (clientId: string, payload: IFocusSignalPayload) => {
		const userId: string = payload.userId;
		const hasFocus: boolean = payload.focus;
```
and `.userId` and `.focus` "do not exist" on `JsonableType`.
I think we preserve the ugly truth (stop hiding it) in this change. We are hoping not to officially support signals any time soon. We can try to improve the typing here, but I think that is another change.
I am using this new stuff with the new LCEDDS feature prototype. Not worried about getting coverage here (currently).

Reply:
Looks like typescript can do enough inferences if we genericize this type. `SignalListener<T> ... payload: Jsonable<T>` will work out. None of our examples need any changes with that; so, decent indicator that it isn't terribly disruptive.